src/resources/elements/primeDesignSystem/_styles.scss

```diff
@@ -1,2 +1,58 @@
 @import "./colors.scss";
-@import "./variables.scss";
\ No newline at end of file
+@import "./variables.scss";
+
+.pInput {
+  font-style: normal;
+  font-weight: normal;
+  font-size: $font-size1;
+  line-height: 23px;
+  border: 1px solid $Border01;
+  background: $BG04;
+  color: $Neutral04;
+  border-radius: $border-radius-normal;
+  outline: none;
+  padding: $spacing-normal;
+  box-sizing: border-box;
+
+  &, input::placeholder {
```

Comment:
font color for filled value is not applied correctly. I guess you should move the decleration into it's underling input element:
```scss
  &>input{
    color: $Neutral04;
    ::placeholder {
      color: $Neutral02;
    }
  }

```

Reply:
Missed that. Thank you! 😄